### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by zelanaakanah** [IAmPicard/StarTrekTimelinesSpreadsheet#254]
**Describe the bug**

I cannot get to the Ships tab. All I get is a blank screen

**To Reproduce**

1. Open App.
2. Click Ships tab

**Expected behavior**

I expected to see a list of my ships.

**Screenshots**

**Additional context**
Tool version: **0.8.2**
Operating system: **win32 x64 (10.0.17134)**

**Comment by smmsttl** [comment]
me too

**Comment by jyhash** [comment]
Agreed. Pretty much anything in the expanded tab is popping up as a blank page, with the exception of  Items.
